### PR TITLE
Add Nodle Solochain

### PR DESCRIPTION
**Pull request by leohar**
## Nodle Solochain ✅ 
<details>
  <summary>Nodle Solochain</summary>

image

</details>

- fixes
- changed Parallel, Integritee, Subsocial, Statemint, Statemine gradients

### CODE REVIEW

File: chains/v2/chains_dev.json [hunk 1]
```diff
@@ -1207,7 +1207,7 @@
                 "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-parallel-heiko"
             }
         },
-        "color": "linear-gradient(315deg, #5314F1 0%, #04E2FF 100%)",
+        "color": "linear-gradient(315deg, #3A55F5 8%, #04E2FF 80%, #3F3F3F 80.01%)",
```
Comment (stepanLav) [inline chains/v2/chains_dev.json:1210]: linear-gradient(315deg, #3A55F5 7.62%, #04E2FF 78.01%, #3F3F3F 78.34%)
<details>
  <summary>Spoiler!</summary>
Screenshot 2022-03-28 at 17 17 22
</details>

File: chains/v2/chains_dev.json [hunk 2]
```diff
@@ -1865,7 +1865,7 @@
                 "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-parallel"
             }
         },
-        "color": "linear-gradient(315deg, #5314F1 0%, #04E2FF 100%)",
+        "color": "linear-gradient(315deg, #5314F1 8%, #04E2FF 80%, #D43079 80.01%)",
```
Comment (stepanLav) [inline chains/v2/chains_dev.json:1868]: linear-gradient(315deg, #5314F1 8.74%, #04E2FF 77.82%, #D43079 77.97%)
<details>
  <summary>Spoiler!</summary>
Screenshot 2022-03-28 at 17 18 19

</details>

File: chains/v2/chains_dev.json [hunk 3]
```diff
@@ -1958,7 +1958,7 @@
                 "url": "https://api.subquery.network/sq/nova-wallet/nova-wallet-statemint"
             }
         },
-        "color": "linear-gradient(315deg, #5B892B 0%, #97D92D 100%)",
+        "color": "linear-gradient(315deg, #23371A 0%, #89C62D 80%, #D43079 80.01%)",
```
Comment (stepanLav) [inline chains/v2/chains_dev.json:1961]: linear-gradient(315deg, #23371A 0%, #89C62D 77.62%, #D43079 77.92%)

<details>
  <summary>Spoiler!</summary>
Screenshot 2022-03-28 at 17 19 11

</details>

File: chains/v2/chains_dev.json [hunk 4]
```diff
@@ -2217,7 +2217,7 @@
             "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/subsocial.json",
             "overridesCommon": true
         },
-        "color": "linear-gradient(315deg, #4A066B 0%, #681391 100%)",
+        "color": "linear-gradient(315deg, #720497 0%, #F50083 80%, #3F3F3F 80.01%)",
```
Comment (stepanLav) [inline chains/v2/chains_dev.json:2220]: linear-gradient(315deg, #720497 0%, #F50083 77.54%, #3F3F3F 77.72%)
<details>
  <summary>Spoiler!</summary>
Screenshot 2022-03-28 at 17 20 00
</details>

File: chains/v2/chains_dev.json [hunk 5]
```diff
@@ -2284,7 +2284,7 @@
             "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/Integritee.json",
             "overridesCommon": true
         },
-        "color": "linear-gradient(315deg, #000C5F 41.07%, #91E3DA 100%)",
+        "color": "linear-gradient(315deg, #000C5F 50%, #5A92AB 80%, #3F3F3F 80.01%)",
```
Comment (stepanLav) [inline chains/v2/chains_dev.json:2287]: linear-gradient(315deg, #000C5F 48.7%, #5A92AB 76.76%, #3F3F3F 76.99%)
<details>
  <summary>Spoiler!</summary>
Screenshot 2022-03-28 at 17 20 52
</details>

File: chains/v2/chains_dev.json [hunk 6]
```diff
@@ -2511,14 +2511,14 @@
             "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/Integritee_solo.json",
             "overridesCommon": true
         },
-        "color": "linear-gradient(315deg, #007568 31.86%, #91E3DA 100%)",
+        "color": "linear-gradient(315deg, #000C5F 50%, #91E3DA 100%)",
```
Comment (stepanLav) [inline chains/v2/chains_dev.json:2514]: linear-gradient(315deg, #000C5F 47.14%, #91E3DA 100%)
<details>
  <summary>Spoiler!</summary>
Screenshot 2022-03-28 at 17 21 34
</details>

**Review by antonkhvorov — APPROVED** [review]

**Pull request merged**